src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,4 +1,21 @@
-// Add the required types and props
-export const UserInfo = () => (
-  <>UserInfo markup</>
-);
+import React from 'react';
+import './UserInfo.scss';
+
+import { User } from '../../types/User';
+
+type Props = {
+  user: User;
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => {
+  const {
+    name,
+    email,
+  } = user;
```

Comment:
```suggestion
  const { name, email } = user;
```
you should put them in separate lines only when you have more then 3 values